tests/openwrt/test_wireless.py

```diff
@@ -179,8 +179,11 @@ def test_parse_multiple_wifi(self):
     }
     _wifi_bridge_uci = """package network
 
+config device 'device_eth0_1'
```

Comment:
This does not look right.
There's a bridge here which does not generate the new device config type, why is that?

Reply:
The previous set_dsa method was changing the self.dsa property for the whole block. So when conversion was happening, self.dsa was False for bridge.
Fixed this in https://github.com/openwisp/netjsonconfig/pull/222/commits/5f94e927351de53caca945b5688d87276747415a